src/topology/instances/ennreal.lean

```diff
@@ -9,12 +9,11 @@ import topology.instances.nnreal
 -/
 
```

Comment:
Do you mind reflowing [the long lines](https://github.com/leanprover-community/mathlib/blob/93b7e6358c89cf17e74895aa4393dd2bf8e58d2b/scripts/copy-mod-doc-exceptions.txt#L2098) in this file?